src/terra-dev-site/contributing/ContributionGuidelines.a.contributing.mdx

```diff
@@ -0,0 +1,71 @@
+# Contribution Guidelines
+
+We love contributions to Terra. To ensure everyone has a good experience contributing we require the following guidelines. If you are new to open source, be sure to check out [How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
+
+If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you intend to work on it so other people don’t accidentally duplicate your effort.
+If somebody claims an issue, but doesn’t follow up for more than two weeks, it’s fine to take over it but you should still leave a comment.
+
+All contributions should follow our [conventions](/contributing/terra-ui/conventions) and [standards](/contributing/terra-ui/component-standards).
+
+If you are unsure on if a contribution would be classified as small or large, leave a comment on the issue and we will let you know!
+
+## Small contributions
+
+For small contributions no additional steps are need. Just opening a PR is generally good enough.
+
+### Examples
+- Bug fixes
+- Variants on components
+- Documentation updates
+- Increasing test coverage
+
+## Large
+
+For larger contributions the Terra team needs to be consulted and included in the design and scope of the contribution.
+
+### Examples
+- A new component
+- Non-passive public api changes
+- Changing the testing framework
+- Major dependency version bumps
+
+### Contributor Expectations
+
+To ensure there are no surprises on PRs and the resulting component, contributors are expected to have regular syncs with the allocated UX and Terra Engineers. This can include periodic meetings and/or attending scrum.
```

Comment:
Should we define regular.  At least weekly?

Reply:
Do we need to? I was ok leaving it open and up to the terra engineer assigned. Depending the the circumstances of the contribution and the complexity.